WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeViewController.swift

```diff
@@ -218,8 +229,9 @@ private extension AddAttributeViewController {
     func configureTextField(cell: TextFieldTableViewCell) {
         let viewModel = TextFieldTableViewCell.ViewModel(text: nil,
                                                          placeholder: Localization.titleCellPlaceholder,
-                                                         onTextChange: { newAttributeName in
-                                                            // TODO: handle new attribute
+                                                         onTextChange: { [weak self] newAttributeName in
+                                                            self?.viewModel.handleNewAttributeNameChange(newAttributeName)
+                                                            self?.enableDoneButton(self?.viewModel.newAttributeName != nil)
```

Comment:
non-blocking idea: the Done button enabled logic could be an observable (like by using `@Published` var) from the view model based on `newAttributeName` (or a boolean that indicates whether there are unsaved changes) then the view controller can subscribe to it to enable/disable the button 😄